src/handwritten/_grpc_time.py

```diff
@@ -13,10 +16,32 @@
 _YS_PER_NS = 10**15
 _YS_PER_FS = 10**9
 
+def convert_to_desired_timezone(expected_time_utc, tzinfo):
```

Comment:
Add a `_` prefix to helper functions to keep them private. 

Please write type hints for helper functions.